technical/20190607-ath9k-test.md

```diff
@@ -0,0 +1,170 @@
+---
+title: OpenWRT AHT9k 5ghz LibreRouter Antenna Tests
+location: Toronto
+attendees: 8
+date: 2018-05-30
```

Comment:
This is not the same date as the filename. The filename should also have an underscore after date.